css/layouts/_footer.scss

```diff
@@ -1,6 +1,8 @@
 section.footer {
 	background: $cl_gray_footer;
 	padding: 0 0 20px;
+	z-index: 100;
```

Comment:
nit: if possible, using as small as possible z-index values is ideal since-- in absence of some extant map of z-index values-- it minimizes the opportunity for hard-to-debug z-index conflicts to arise.  It is much easier to know if a display issue of a container is z-index related if incrementing by small values than large ones (e.g. if a `div` is being hidden unexpectedly and you are unsure if the reason is a z-index conflict or not, bumping the `div`'s z-index to 100 will definitely reveal the answer when everything in the site CSS in z-index < 10 vs. if z-indexes are of arbitrarily large values and therefore your arbitrarily large z-index test-value may or may not exceed the z-index of some unknown element that could be causing a conflict).

Reply:
Moved down to 1 and still shows the desired behaviour; thanks for catching that, Joe.